docs/src/pages/components/visuallyhidden/react.mdx

```diff
@@ -1,25 +1,29 @@
-import { Button, IconDone, VisuallyHidden } from '@aws-amplify/ui-react';
-import { Example } from '@/components/Example';
+import { Button, IconDone, VisuallyHidden, View } from '@aws-amplify/ui-react';
+import { Example, ExampleCode } from '@/components/Example';
+import { DefaultVisuallyHiddenExample } from './examples';
 
 `VisuallyHidden` is a common techinique used in web accessibility to hide content from the visual client, but keep it readable for screen readers.
 
 ## Usage
 
 Import the `VisuallyHidden` primitive and styles.
```

Comment:
```suggestion
Import the `VisuallyHidden` primitive.
```

Reply:
To be consistent with the example file and we've talked about removing this as users only need to import style once in their APPs